doc/user/content/releases/v0.32.md

```diff
@@ -49,3 +46,20 @@ This version of Materialize is not yet released.
   [AWS PrivateLink connections](/sql/create-connection/#aws-privatelink),
   which establish links to
   [AWS PrivateLink](https://aws.amazon.com/privatelink/) services.
+
+## Patch releases
+
+### v0.32.1
+
+* Stabilize the performance of ad hoc `SELECT` statements against unindexed
+  objects in large clusters {{% gh 16090 %}}.
+
+### v0.32.2
+
+* Fix a memory leak caused by lingering read handles when dropping materialized
+  views or indexes {{% gh 16127 %}}.
```

Comment:
Not sure how to phrase this one in the way that best captures the user impact. It was, technically, a memory leak, but the memory wasn't the big problem; it was also a storage leak, in that it kept a bunch of blobs on S3 and in CockroachDB that didn't need to be kept around; eventually there'd be so many blobs piled up that other queries against the view/index would be sloooooow. And that latter point was the thing users were actually feeling. 🤷🏽 

If you want to go all in on the user impact:

> Fix a bug that would cause query performance on unindexed objects to slowly degrade after each release.